### ISSUE TITLE
Any way to dynamic create logs in runtime?

### ISSUE THREAD

**Issue by jvalenciag** [SergiusTheBest/plog#46]
Is possible to create separate logs in runtime, not based on a template?

**Comment by SergiusTheBest** [comment]
Could you provide an example?

**Comment by jvalenciag** [comment]
As showed in your MultiInstance example, "SecondLog" is passed as a template parameter to instantiate a second log which makes impossible to create logs based on non constant variables, dynamically in runtime.

````
plog::init<SecondLog>(plog::debug, "MultiInstance-second.txt"); // Initialize the 2nd logger instance.
````
Currently I need a log instance per object so each object write logs to different files.

**Comment by SergiusTheBest** [comment]
I see. Currently it's not possible as log instances must be known at compile time.

**Comment by shreks7** [comment]
Thank you for this amazing log library. This feature would be really useful - different instances can be created in runtime by passing in dynamic InstanceId.

**Comment by SergiusTheBest** [comment]
This feature is implemented in #178.
